components/kyma-environment-broker/internal/reconciler/contract.go

```diff
@@ -28,6 +28,7 @@ type Configuration struct {
 }
 
 type Components struct {
+	URL           string          `json:"URL"`
```

Comment:
just for my understanding, what was the reason of implementing the contract here as opposed to [importing from reconciler directly](https://github.com/kyma-incubator/reconciler/blob/587b592eaa2c82d7d923ffdf0297f04fe00dfae7/pkg/keb/model_gen.go#L74-L79)?

Reply:
I don't remember....